packages/proto-rpc/src/RpcCommunicator.ts

```diff
@@ -0,0 +1,278 @@
+import * as Err from './errors'
+import { ErrorCode } from './errors'
+import {
+    ClientTransport,
+    ResultParts,
+    ProtoRpcOptions,
+    Event as DhtTransportClientEvent
+} from './ClientTransport'
+import {
+    RpcMessage,
+    RpcResponseError
+} from './proto/ProtoRpc'
+import { Empty } from './proto/google/protobuf/empty'
+import { CallContext, ServerRegistryEvent as ServerRegistryEvent, Parser, Serializer, ServerRegistry } from './ServerRegistry'
+import { EventEmitter } from 'events'
+import { DeferredState } from '@protobuf-ts/runtime-rpc'
+import { ServerCallContext } from '@protobuf-ts/runtime-rpc'
+import { Logger } from './Logger'
+
+export enum StatusCodes {
```

Comment:
I think we've tend to favor singular in names
```suggestion
export enum StatusCode {
```